locales/en/translations.json

```diff
@@ -1009,6 +1009,180 @@
       "zambian" : "Zambian",
       "zimbabwean" : "Zimbabwean"
     },
+    
+    "currency": {
+        "GBP" : "GBP",
+        "USD" : "USD",
```

Comment:
do we need translations for the currencies as they should be the same in all languages

Reply:
The text should be a description with a code "Afghani (AFN)" so updated the translations to include this. Judging from the api it looks like it is the code such as "AFN" that we want to send to kept those values the same.

Reply:
To answer the question this means we will be needing the translations.